Sources/Basics/FileSystem+Extensions.swift

```diff
@@ -118,8 +118,19 @@ extension FileSystem {
     public func readFileContents(_ path: AbsolutePath) throws -> Data {
         return try Data(self.readFileContents(path).contents)
     }
-
+    
+    public func readFileContents(_ path: AbsolutePath) throws -> String {
+        guard let string = try String(data: self.readFileContents(path), encoding: .utf8) else {
```

Comment:
Would it make sense to use `String(decoding: self.readFileContents(path), as: UTF8.self)` here, at least optionally as a non-throwing version?  It substitutes replacements but maybe we want to have the stricter checking for this particular kind of file.